examples/README.md

```diff
@@ -16,38 +16,59 @@ AMOP（Advanced Messages Onchain Protocol）即链上信使协议，旨在为联
 
 ## 单播案例
 
-**单播** 就是节点从监听同一 Topic 的多个订阅者中随机抽取一个订阅者转发消息
+**单播** 指的是节点从监听同一 Topic 的多个订阅者中随机抽取一个订阅者转发消息
 
--   启动 AMOP 服务端：
+-   运行 AMOP 消息发布者：go run examples/amop/unicast_pub/publisher.go [endpoint] [topic]
```

Comment:
Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5979849)